File: src/2020/1/js/1.2.js
fs = require("fs");
fs.readFile("src/2020/1/inputs.txt", "utf8", function (err, data) {
  err && console.log(err);
  const inputs = data.split("\n").map(Number);
  console.time("exec");
  loop1: for (let i = 0; i < inputs.length; i++) {
    for (let j = 0; j < inputs.length - 1; j++) {
      for (let k = 0; k < inputs.length - 2; k++) {
        if (inputs[i] + inputs[j] + inputs[k] === 2020) {
          console.log(inputs[i] * inputs[j] * inputs[k]);
          console.timeEnd("exec");
          break loop1;
        }
      }
    }
  }
});

File: src/2020/1/python/1.2.py
from itertools import combinations
import time

with open("src/2020/1/inputs.txt", "r") as f:
    inputs = list(map(int, f.read().splitlines()))
    tic = time.perf_counter()

    for x, y, z in combinations(inputs, 3):
        if x + y + z == 2020:
            print(x * y * z)
            break

    toc = time.perf_counter()
    print(f"Exec in: {toc - tic:0.4f}s")
